app/javascript/components/DownloadButton.js

```diff
@@ -108,17 +110,35 @@ function DownloadCommandContainer(props) {
  */
 export default function DownloadButton(props) {
 
+  const searchContext = useStudySearchContext()
+  const userContext = useUserContext()
+
+  const matchingAccessions = searchContext.results.matchingAccessions || [];
+
+  /**
+   * Reports whether Download button be active,
+   * i.e. user is signed in and has search results
+   */
+  function shouldBeActive() {
+    return userContext.accessToken !== '' && matchingAccessions.length > 0;
+  }
+
```

Comment:
I don't see why this is state -- it's an entirely derived value

Reply:
Addressed in b72b604.